Xamarin.Forms.Core/Entry.cs

```diff
@@ -95,5 +103,17 @@ static void OnTextChanged(BindableObject bindable, object oldValue, object newVa
 
 			entry.TextChanged?.Invoke(entry, new TextChangedEventArgs((string)oldValue, (string)newValue));
 		}
+
+		public static void OnIsFocusedChanged (object sender, object oldValue, object newvalue)
+		{
+			OnIsFocusedChanged ((VisualElement)sender, (bool)oldValue, (bool)newvalue);
+		}
+
+		public static void OnIsFocusedChanged (VisualElement sender, bool oldValue, bool newValue)
+		{
+			if (newValue)
+				sender.Focus ();
+			sender.SetValue (IsFocusedProperty, false);
+		}
```

Comment:
Could you move those Entry changes to a different PR?
This is a huge change, it certainly requires changes on each platforms. It'd require tons of UI and unit tests to make sure that
- only one element is focused (i.e. has the IsFocused property) at any time
- it works well in even the more complex solutions